src/main/java/com/dev/cinema/util/HibernateUtil.java

```diff
@@ -0,0 +1,24 @@
+package com.dev.cinema.util;
+
+import com.dev.cinema.exeptions.DataProcessingException;
+import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
+
+public class HibernateUtil {
+    private static SessionFactory sessionFactory = initSessionFactory();
+
+    private HibernateUtil() {
+    }
+
+    private static SessionFactory initSessionFactory() {
+        try {
+            return new Configuration().configure().buildSessionFactory();
+        } catch (Exception e) {
+            throw new DataProcessingException("Error creating Session Factory", e);
```

Comment:
```suggestion
            throw new RuntimeException("Error creating Session Factory", e);
```

this exception has a different nature

Reply:
done